### DISCUSSION TITLE
Need more number converters, need more switch options.

### DISCUSSION THREAD

**Discussion by Ferniclestix** [WASasquatch/was-node-suite-comfyui#131] [Ideas]
Currently trying to do some math stuff using WAS nodes and keep finding myself unable to do certain conversions like Int to number which forces me to do this stuff, 
![Capturesss](https://github.com/WASasquatch/was-node-suite-comfyui/assets/128430314/1b10edd3-fc05-49e6-9c16-388f6f92ab69)
 
The number operation node is amazing but almost nothing will go into it which drastically reduces its usefulness.

Second thing, the current switch options are ok but we need switches that go the other way too as well as the option to end at the switch if a certain input is selected. selecting an output is just as useful as selecting an input. and could allow use to switch on and off parts of our workflow as needed without disconnecting noodles.

**Comment by WASasquatch** [comment]
`INT`, `FLOAT` is reserved by the front-end, you can't use it for a inputs. The nodes won't even show with `INT` or  `FLOAT` as a input, even if you use `{"forceInput":True}`. I've already been through this and tried to add them. Litegraph is a pretty terrible system for ComfyUI and it's needs imo. 

Switches are just this or that, there is no real "reverse" as the output is controlled by the input, and the bool allows you to change this input value to switch it. 

And I'm not sure what you mean exactly by "end at switch"? We have to output something or ComfyUI will error out and stop all execution.

**Reply by Ferniclestix** [reply]
:P all g, my programmer friend stares at me and sighs alot too before carefully explaining that no this thing is in fact not as easy as blah.
